src/main/scala/com/github/drmarjanovic/app/api/model/ErrorResponseData.scala

```diff
@@ -0,0 +1,9 @@
+package com.github.drmarjanovic.app.api.model
+
+import zio.json._
+
+final case class ErrorResponseData(body: List[String])
```

Comment:
It can be value class.

Reply:
Actually, I tried that, but since `ErrorResponse` (`ErrorResponse(errors: ErrorResponseData) extends AnyVal`) is already a value class, it's not allowed to convert this to the value class.